R/seurat.R

```diff
@@ -5872,6 +5872,7 @@ UpdateDimReduction <- function(old.dr, assay) {
       cell.embeddings = as(object = cell.embeddings, Class = 'matrix'),
       feature.loadings = as(object = feature.loadings, Class = 'matrix'),
       assay.used = assay,
+      global = FALSE,
```

Comment:
```suggestion
      global = grepl(pattern = 'tsne|umap', x = i, ignore.case = TRUE),
```

Reply:
Thanks for taking a look at this! Regarding your suggestion, I think this is already being taken care of in lines 1050-1053 in seurat.R? That is why the error only showed for pca but not for the other embeddings. But I may be wrong!

https://github.com/satijalab/seurat-object/blob/ee82ea7e861a680e69704c00b4288fcd3f9c9f25/R/seurat.R#L1050-L1053

Reply:
Yep, you're absolutely right; thanks for pointing that out. This can be accepted without my suggestion. Thanks again for taking care of this!